akka-docs/src/main/paradox/common/circuitbreaker.md

```diff
@@ -80,9 +80,9 @@ Java
 
 ### Future & Synchronous based API
 
-Once a circuit breaker actor has been initialized, interacting with that actor is done by either using the Future based API or the synchronous API. Both of these APIs are considered `Call Protection` because whether synchronously or asynchronously, the purpose of the circuit breaker is to protect your system from cascading failures while making a call to another service. In the future based API, we use the `withCircuitBreaker` which takes an asynchronous method (some method wrapped in a `Future`), for instance a call to retrieve data from a database, and we pipe the result back to the sender. If for some reason the database in this example isn't responding, or there is another issue, the circuit breaker will open and stop trying to hit the database again and again until the timeout is over.
+Once a circuit breaker actor has been initialized, interacting with that actor is done by either using the Future based API or the synchronous API. Both of these APIs are considered `Call Protection` because whether synchronously or asynchronously, the purpose of the circuit breaker is to protect your system from cascading failures while making a call to another service. In the future based API, we use the @apidoc[withCircuitBreaker](CircuitBreaker) {scala="#withCircuitBreaker[T](body:=%3Escala.concurrent.Future[T]):scala.concurrent.Future[T]" java="#callWithCircuitBreaker(java.util.concurrent.Callable)"} which takes an asynchronous method (some method wrapped in a @scaladoc[Future](scala.concurrent.Future)), for instance a call to retrieve data from a database, and we pipe the result back to the sender. If for some reason the database in this example isn't responding, or there is another issue, the circuit breaker will open and stop trying to hit the database again and again until the timeout is over.
```

Comment:
Am I right in the paragraph? I'm not sure about right java method for `withCircuitBreaker`. Should I use `Callable` here?

> some method wrapped in a @scaladoc[Future]

Reply:
I think it should be `callWithCircuitBreakerCS` for Java, and then alternative text for Java mentioning/linking to `java.util.concurrent.CompletionStage` in the JDK instead.